src/Url.js

```diff
@@ -52,7 +52,7 @@ const Url = React.createClass({
 
   onClick(event) {
     event.preventDefault();
-    ActionCreator.navigateTo(this.props);
+    Store.dispatch(ActionCreator.navigateTo(this.props));
```

Comment:
Yeah just double checking Redux makes a big distinction between "smart" components which talk to Redux directly & "dumb" components which update themselves from their parent components usually. So I'm going to have to go back and think that through because atm they all talk to the Store/Dispatcher. Could maybe add the `react-redux` package (since subscribe/dispatch etc are supposed to be a bit low level)?

Otherwise though, according to the guides, Components fetch/update/listen to data via the Store but there must be only one Store per app. So I am (fairly) sure the data is still essentially unidirectional despite the fact its calling both dispatch and getState on the same Store.